icomoon/style.css

```diff
@@ -0,0 +1,42 @@
+@font-face {
+  font-family: 'icomoon';
+  src:  url('fonts/icomoon.eot?37nhmb');
+  src:  url('fonts/icomoon.eot?37nhmb#iefix') format('embedded-opentype'),
+    url('fonts/icomoon.ttf?37nhmb') format('truetype'),
+    url('fonts/icomoon.woff?37nhmb') format('woff'),
+    url('fonts/icomoon.svg?37nhmb#icomoon') format('svg');
```

Comment:
We don't need so many obsolete font formats. Leave only woff.

Reply:
In the next version of project I left only woff format.
But in this part after your notice, I've done the same.